cmd/deployment-check/deployment-check.yaml

```diff
@@ -10,13 +10,17 @@ spec:
   podSpec:
     containers:
     - name: deployment
-      image: kuberhealthy/deployment-check:v1.6.2
+      image: kuberhealthy/deployment-check:v1.7.2
       imagePullPolicy: IfNotPresent
       env:
         - name: CHECK_DEPLOYMENT_REPLICAS
           value: "4"
         - name: CHECK_DEPLOYMENT_ROLLING_UPDATE
           value: "true"
+        - name: TOLERATION_VALUE
```

Comment:
Having this in the example assumes that everyone has these taints & node labels -- maybe put them under a new section in the readme?

Reply:
removed values for these and left as blank which should be default.